interactions.js

```diff
@@ -1,9 +1,53 @@
-$(window).scroll(function () {
-  if ( $(this).scrollTop() > 200 && !$('header').hasClass('open') ) {
-    $('header').addClass('open');
-    $('header').slideDown();
-   } else if ( $(this).scrollTop() <= 200 ) {
-    $('header').removeClass('open');
-    $('header').slideUp();
-  }
-});
+var hover = (function()
+{
+    function mouseMoveF(e)
+    {
+        if(!this.show && this.classList.contains('hidden'))
+        {
+            this.classList.remove('hidden');
+            return;
+        }
+
+        if(this.show.classList.contains('hidden'))
```

Comment:
please fix your functions' first lines to this styling: function() {

This is the more standard way.